ACLWeb.js

```diff
@@ -193,14 +193,14 @@ function getSearchResults(doc, url) {
 	let items = {};
 	if (url.includes('/search/')) {
 		// e.g. https://www.aclweb.org/anthology/search/?q=foo+bar
-		papers = ZU.xpath(doc, '//div[@class = "gsc-wrapper"]//td//a[contains(@class, "gs-title")]');
+		papers = ZU.xpath(doc, '//div[contains(@class, "gsc-webResult")]//div[@class = "gs-title"]/a');
```

Comment:
Did you change to `@class=` instead of only `contains` by a special reasons? I would prefer to use conains for class attributes, as this is more stable also when another class is added in the future.

Reply:
No special reason, but good point. Have changed back